autogpt_platform/backend/backend/util/json.py

```diff
@@ -30,3 +31,15 @@ def loads(data: str, *args, target_type: Type[T] | None = None, **kwargs) -> Any
     if target_type:
         return type_match(parsed, target_type)
     return parsed
+
+
+def validate_jsonschema(schema: dict[str, Any], data: dict[str, Any]) -> str | None:
```

Comment:
```suggestion
def validate(schema: dict[str, Any], data: dict[str, Any]) -> str | None:
```

since it doesn't validate the schema, it validates the data.

Reply:
Updated